<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Administrator/Pictures/头像/墨小蛙.jpg</normaloff>C:/Users/Administrator/Pictures/头像/墨小蛙.jpg</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>371</width>
      <height>500</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">rgb:(255, 170, 0);</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1051</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Information</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>510</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="NameLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NameInput">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="CommentInput">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>330</y>
         <width>180</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="BoyRadio">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>65</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Boy</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="GirlRadio">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>120</y>
         <width>65</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Girl</string>
       </property>
      </widget>
      <widget class="QLabel" name="GenderLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CommentLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>330</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ProfilePwd">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="SkiilPwd">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="NaturePwd">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>240</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="ProfileLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>180</y>
         <width>50</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Profile:</string>
       </property>
      </widget>
      <widget class="QLabel" name="SkillLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>50</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Skill:</string>
       </property>
      </widget>
      <widget class="QLabel" name="NatureLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>240</y>
         <width>50</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Nature:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PhoneLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PerformancePwd">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>270</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="PerformanceLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>270</y>
         <width>80</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Performance:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Commit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>430</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Cancel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>430</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PhoneInput">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <zorder>NameLabel</zorder>
      <zorder>CommentInput</zorder>
      <zorder>BoyRadio</zorder>
      <zorder>GirlRadio</zorder>
      <zorder>GenderLabel</zorder>
      <zorder>CommentLabel</zorder>
      <zorder>ProfilePwd</zorder>
      <zorder>SkiilPwd</zorder>
      <zorder>NaturePwd</zorder>
      <zorder>ProfileLabel</zorder>
      <zorder>SkillLabel</zorder>
      <zorder>NatureLabel</zorder>
      <zorder>PhoneLabel</zorder>
      <zorder>PerformancePwd</zorder>
      <zorder>PerformanceLabel</zorder>
      <zorder>Commit</zorder>
      <zorder>Cancel</zorder>
      <zorder>NameInput</zorder>
      <zorder>PhoneInput</zorder>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>431</width>
        <height>510</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="autoScrollMargin">
       <number>100</number>
      </property>
     </widget>
     <zorder>tableView</zorder>
     <zorder>groupBox</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Comment</string>
     </attribute>
     <widget class="QLineEdit" name="PasswordInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ShowButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
     <widget class="QTableView" name="CommentTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>800</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>NameInput</tabstop>
  <tabstop>PhoneInput</tabstop>
  <tabstop>BoyRadio</tabstop>
  <tabstop>GirlRadio</tabstop>
  <tabstop>ProfilePwd</tabstop>
  <tabstop>SkiilPwd</tabstop>
  <tabstop>NaturePwd</tabstop>
  <tabstop>PerformancePwd</tabstop>
  <tabstop>CommentInput</tabstop>
  <tabstop>Commit</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
